roles/cds/tasks/main.yml

```
@@ -0,0 +1,26 @@
+- name: install cds package
+  ansible.builtin.apt:
+    pkg: icpc-host-cds
+    state: present
+
+- name: Check if cds is installed
+  ansible.builtin.stat:
+    path: /home/deboer/wlp/bin/server
+  register: cds
+
+- name: Download and install the cds
+  ansible.builtin.unarchive:
+    src: https://github.com/icpctools/icpctools/releases/download/v{{ cds_version }}/wlp.CDS-{{ cds_version }}.zip
+    dest: /home/deboer
+    remote_src: true
+    owner: root
+    group: root
+  when: not cds.stat.exists
+  register: cds_installed
+
+- name: start the cds
+  ansible.builtin.service:
+    name: websphere
+    state: restarted
+    enabled: true
+  when: cds_installed.changed
```

```suggestion
  notify: restart cds

handlers:
- name: restart cds
  ansible.builtin.service:
    name: websphere
    state: restarted
    enabled: true
```
This does not work directly as you need to move the handler somewhere else but I hope you get what I'm suggesting

---

changed to notify: restart cds,  moved handleres to handlers/main.yml
also setup the repo for wlpall to work so websphere service works.